ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Nagios_Plugins/tasks/nagios_CentOS.yml

```diff
@@ -0,0 +1,129 @@
+#########################################################################################
+# AdoptOpenJDK - Ansible Playbook to install Nagios plugins on CentOS 7 on x86 hardware #
+#########################################################################################
+
+###########################################################################
+# License Information:                                                    #
+# Nagios core and its plugins are lincesed under GPL                      #
+# For more information please see: https://www.nagios.com/legal/licenses/ #
+###########################################################################
+
+  - debug: msg='Installing Nagios plugins'
+  ###############
+  # Nagios user #
+  ###############
+  - name: Create Nagios user
```

Comment:
Can any of these tasks be extracted out into the [main.yml](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Nagios_Plugins/tasks/main.yml)? I imagine that many of these are the same commands for each Linux distro? Thoughts @geraintwjones @bblondin

Reply:
Possibly, assuming there are tasks that are common to all platforms. I'll wait to see @bblondin's thoughts, but I'd rather refactor this once we've got all platform playbooks written, and we know exactly what is common.

Reply:
I have compared the 4 existing Linux playbooks (one being the CentOS one that's part of this pull request) and have determined that there are only 4 common tasks, all of which are to do with creating a Nagios user.

These are...
- Set authorized key for Nagios  user
- Set Nagios user account policy - expire date
- Set Nagios user account policy - max days

So maybe we could move these out into main.yml.

However, I'd like @bblondin to confirm that the order in which the tasks are executed isn't important. In the CentOS and RHEL playbooks, creating the nagios user is the first thing done, but in SLES and Ubuntu it is not.

Reply:
I like this idea, it would clean things up. 
However at this stage I would like to have it working for all major OS first, then perform the clean up

Reply:
RHEL and CentOS will likely be identical

Reply:
Looks like I'm now able to squash and merge. However, since these are my changes, I feel that would be the wrong thing to do, that somebody else should do it.

Reply:
@gdams We're going to do te refactoring separately, under a different issue. Could you approve this one please?_